### PR TITLE
DOCS-#4545: Add socials to README

### LINKED ISSUE #4545: Link to Modin Slack in Readme and docs
Make it more obvious that there's a place people can come to with questions

### PR DESCRIPTION
**Pull request by jeffreykennethli**
Signed-off-by: jeffreykennethli <jkli@ponder.io>

Adds pepy link for total Modin downloads, moves Slack and Discourse to a separate table so users know which socials they should join if so willing, and add a link to the Modin project Twitter.

The rationale for Slack -> Dev Community & Support and Discourse -> Community Questions is that Slack allows us to offer dev support in real-time, and surface updates (as does Twitter) like new releases. Discourse is good as a repo of questions people have, like StackOverflow.

- [ ] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/development/contributing.html#commit-message-formatting)
- [ ] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [ ] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [ ] signed commit with `git commit -s` 
- [ ] Resolves #4545  
- [ ] tests added and passing
- [ ] module layout described at `docs/development/architecture.rst` is up-to-date 
- [ ] added (Issue Number: PR title (PR Number)) and github username to release notes for next major release

### CODE REVIEW

**Comment by dorisjlee** [comment]
Thanks @jeffreykennethli! Can we put the grid above the list of badges? Both the badges and the grid should be centered (like the header up top).

**Comment by jeffreykennethli** [comment]
Moved the table above the badges, and centered the badges. Unfortunately, Github-flavored markdown doesn't allow centering the entire table, as the table is rendered as "filled" by default, see https://stackoverflow.com/a/44233457/19027728

Some options are:
* leave the table as is
* reformat the table so that each row contains a badge, purpose, description, etc. to fill the width
* remove the table, and replace it with a short line describing ways to connect with Modin Developers. Something like "Join the Modin community on Slack for active dev support and updates." Then combine the Discourse and Twitter badges with the remaining badges.

**Comment by RehanSD** [comment]
> Moved the table above the badges, and centered the badges. Unfortunately, Github-flavored markdown doesn't allow centering the entire table, as the table is rendered as "filled" by default, see https://stackoverflow.com/a/44233457/19027728
> 
> 
> 
> Some options are:
> 
> * leave the table as is
> 
> * reformat the table so that each row contains a badge, purpose, description, etc. to fill the width
> 
> * remove the table, and replace it with a short line describing ways to connect with Modin Developers. Something like "Join the Modin community on Slack for active dev support and updates." Then combine the Discourse and Twitter badges with the remaining badges.

I think it may be possible to do if you use HTML instead of markdown, like we do for the badges at the top of the readme? The link you sent also seems to indicate that that may be possible!

**Comment by jeffreykennethli** [comment]
> I think it may be possible to do if you use HTML instead of markdown, like we do for the badges at the top of the readme? The link you sent also seems to indicate that that may be possible!

Yup! Was able to do this after all.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
 <p align="center"><a href="https://modin.readthedocs.io"><img width=77% alt="" src="https://github.com/modin-project/modin/blob/3d6368edf311995ad231ec5342a51cd9e4e3dc20/docs/img/MODIN_ver2_hrz.png?raw=true"></a></p>
 <h2 align="center">Scale your pandas workflows by changing one line of code</h2>
 
+<div align="center">
+
+Dev Community & Support (<1 day response) | Forums | Socials |
```
Comment (dorisjlee) [inline README.md:6]: I don't know if we can promise <24 hr response, esp with timezones, etc.

```suggestion
Dev Community & Support | Forums | Socials |
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+|:---: | :---: | :---: |
+| <a href='https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA'><img src='https://img.shields.io/static/v1?label=chat&logo=slack&message=Slack&color=brightgreen' alt='Slack' height=28/></a>     | <a href="https://discuss.modin.org"><img alt="" src="https://img.shields.io/badge/discourse-forum-purple.svg?logo=discourse&logoColor=white" height=28></a> [![Stack Overflow](https://img.shields.io/badge/-Stackoverflow-FE7A16?style=for-the-badge&logo=stack-overflow&logoColor=white)](https://stackoverflow.com/questions/tagged/modin) | [![Twitter](https://img.shields.io/twitter/follow/modin_project?style=social&logo=twitter)](https://twitter.com/modin_project) |
```
Comment (dorisjlee) [inline README.md:8]: Do some of the buttons appear oversized for you compared to the Twitter one?
Screen Shot 2022-06-08 at 8 01 47 PM

Comment (dorisjlee) [reply README.md:8]: Should we moves docs up here too or keep just the socials?

Comment (dorisjlee) [reply README.md:8]: I could imagine the grid being any links that are "actionable" while the badges are more around metrics.

**Review by dorisjlee — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jeffreykennethli) [reply README.md:8]: I considered that ordering as well. I will add Docs to the table and we can compare.

**Comment by mvashishtha** [comment]
@jeffreykennethli  Thank you for doing this! Should we get rid of the line between the Modin slogan and the links table? 
Screen Shot 2022-06-09 at 4 53 22 PM

**Comment by jeffreykennethli** [comment]
> @jeffreykennethli Thank you for doing this! Should we get rid of the line between the Modin slogan and the links table?

Sure @mvashishtha! It would involve changing a `h2` to `h3` which removes the line and makes the text slightly (imo barely perceptible) smaller. Is that all right? See: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#h2 for reference.

**Comment by mvashishtha** [comment]
> > @jeffreykennethli Thank you for doing this! Should we get rid of the line between the Modin slogan and the links table?
> 
> Sure @mvashishtha! It would involve changing a `h2` to `h3` which removes the line and makes the text slightly (imo barely perceptible) smaller. Is that all right? See: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#h2 for reference.

I prefer h2 then, because I want the h2 text size.

**Review by RehanSD — CHANGES_REQUESTED** [review]
Looks good to me, but quick, minor, question - why is the name + logo not consistent across the discourse + slack buttons?
image
in the picture above, the slack button has chat near the logo, and slack on the right side in the green, but the discourse button has discourse near the logo, and forum on the right side in the purple. I think it would be good to standardize this across buttons, or just get rid of it entirely (since having forum in the button and the table column header is slightly repetitive). If the second option is possible, and we do end up going with it, I think maybe just having the whole button be green and having the Slack logo + Slack for the slack button, and having the discourse button be purple and having the discourse logo + discourse would look nice!

Example Mockup:
image

**Comment by jeffreykennethli** [comment]
@RehanSD I agree with your points about "chat" and "forum" being unnecessary. I changed the Slack badge to one that doesn't have the "chat" text, but didn't find a Discourse badge that looked like your mockup. Instead I used a badge from here: https://shields.io/category/chat (you can see Discourse has a few badge options, I went with status). We can also create our own Discourse badge if you think it's worth it.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
 <h2 align="center">Scale your pandas workflows by changing one line of code</h2>
 
+<div align="center">
+
+| <h3>Dev Community & Support</h3> | <h3>Forums</h3> | <h3>Socials</h3> | <h3>Docs</h3> |
+|:---: | :---: | :---: | :---: |
+| [![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA) | <a href="https://discuss.modin.org"><img alt="" src="https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscuss.modin.org" height=28></a> [![Stack Overflow](https://img.shields.io/badge/-Stackoverflow-FE7A16?style=for-the-badge&logo=stack-overflow&logoColor=white)](https://stackoverflow.com/questions/tagged/modin) | <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/modin_project?style=social" height=28 align="center"> | <a href="https://modin.readthedocs.io/en/latest/?badge=latest"><img alt="" src="https://readthedocs.org/projects/modin/badge/?version=latest" height=28 align="center"></a> |
```
Comment (devin-petersohn) [inline README.md:8]:
```suggestion
| [![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA) | [![Stack Overflow](https://img.shields.io/badge/-Stackoverflow-FE7A16?style=for-the-badge&logo=stack-overflow&logoColor=white)](https://stackoverflow.com/questions/tagged/modin) | <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/modin_project?style=social" height=28 align="center"> | <a href="https://modin.readthedocs.io/en/latest/?badge=latest"><img alt="" src="https://readthedocs.org/projects/modin/badge/?version=latest" height=28 align="center"></a> |
```

**Review by devin-petersohn — COMMENTED** [review]
I prefer to route people to the Slack instead of discourse. What do you think?

**Review by RehanSD — DISMISSED** [review]
LGTM! Should we add something to the release notes for this?

**Review by RehanSD — APPROVED** [review]
LGTM!

File: README.md [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
+<div align="center">
+
+| <h3>Dev Community & Support</h3> | <h3>Forums</h3> | <h3>Socials</h3> | <h3>Docs</h3> |
+|:---: | :---: | :---: | :---: |
+| [![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA) | [![Stack Overflow](https://img.shields.io/badge/-Stackoverflow-FE7A16?style=for-the-badge&logo=stack-overflow&logoColor=white)](https://stackoverflow.com/questions/tagged/modin) | <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/modin_project?style=social" height=28 align="center"> | <a href="https://modin.readthedocs.io/en/latest/?badge=latest"><img alt="" src="https://readthedocs.org/projects/modin/badge/?version=latest" height=28 align="center"></a> |
+
+</div>
+
 <p align="center">
-<a href="https://discuss.modin.org"><img alt="" src="https://img.shields.io/badge/discourse-forum-purple.svg?logo=discourse&logoColor=white" align="center"></a>
```
[the remark this answers is not in the document]
Comment (YarShev) [reply README.md:5]: If it is intended not to bloat the table, that is okay to me. Just to be informed please point me at the related discussion or comment.

**Review by YarShev — APPROVED** [review]

**Pull request merged**